tests/languages/smalltalk/string_character_feature.test

```diff
@@ -0,0 +1,13 @@
+$'
+'foobar'
+
+----------------------------------------------------
+
+[
+	["character", "$x"],
+	["string", "'foobar'"]
+]
+
+----------------------------------------------------
+
+xChecks for strings.

```

Comment:
```suggestion
Checks for strings.
```

Reply:
thx.  I added that 'x' while experimenting trying to work out how the build/test worked, and forgot.